apps/builder/app/builder/features/sidebar-left/help-popover.tsx

```diff
@@ -0,0 +1,94 @@
+import {
+  Button,
+  Flex,
+  Popover,
+  PopoverContent,
+  PopoverTrigger,
+  theme,
+} from "@webstudio-is/design-system";
+import {
+  DiscordIcon,
+  GithubIcon,
+  TwitterIcon,
+  WebstudioWhiteIcon,
+  YoutubeIcon,
+} from "@webstudio-is/icons";
+import { useState, type ComponentProps } from "react";
+
+const PopoverItemButton = (
+  props: ComponentProps<typeof Button> & { background?: string; url?: string }
+) => (
+  <Button
```

Comment:
Since all of these are basically links, I think you could render as="a" and put href and target="_blank"

Reply:
I thought of it too. Will add that.